hudi-common/src/test/java/org/apache/hudi/common/data/TestHoodieListData.java

```diff
@@ -72,4 +73,23 @@ public void testGetNumPartitions() {
         IntStream.rangeClosed(0, 100).boxed().collect(Collectors.toList()));
     assertEquals(1, listData.getNumPartitions());
   }
+
+  @Test
+  public void testIsEmpty() {
+    // HoodieListData bearing eager execution semantic
+    HoodieData<Integer> listData = HoodieListData.eager(
+            IntStream.rangeClosed(0, 100).boxed().collect(Collectors.toList()));
+    assert !listData.isEmpty();
```

Comment:
use `assertTrue()` or `assertFalse()`?

Reply:
OK, IDEA grammar recommends using assert when assertTrue, but if our project is accustomed to using assertTrue, I will modify it now.

Reply:
@yihua bro, Is the modification initiated as a new commit or merged into one commit?

Reply:
Ok, no problem